block/manager.go

```diff
@@ -60,6 +60,7 @@ type Manager struct {
 	daHeight uint64
 
 	HeaderCh chan *types.SignedHeader
+	BlockCh  chan *types.Block
```

Comment:
you need to initialize the channel?

Reply:
also need to do `m. BlockCh <- &block` in `publishBlock`?

Reply:
Added